lib/encryption_utils.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
```

Comment:
For consistency, please name this file `lib/encryptionUtils.js`, Mongoose camelCases file names

Reply:
Does this apply to test files too?

Reply:
Yes